charts/perses/values.yaml

```diff
@@ -150,7 +150,7 @@ persistence:
   enabled: false
 
   # -- Specify the `storageClass` used to provision the volume
-  storageClass: ""
+  storageClass: "default"
```

Comment:
I think we don't need to set default as default, if we omit it k8s will handle it as default SC.

Reply:
great catch. fixed in https://github.com/perses/helm-charts/pull/28/commits/890b126bdd2bc1752560e3e1f784d0d479a428c7